packages/super-components/components/msc-tile/src/msc-tile/msc-tile.scss

```diff
@@ -0,0 +1,45 @@
+.msc-tile-wrapper {
+  display: block;
+  text-decoration: none;
+  height: 100%;
+
+  .msc-ods-tile {
+    width: 100%;
+    height: 100%;
+  }
+
+  .tile-content {
+    display: flex;
+    flex-direction: column;
+
+    .tile-img {
+      height: auto;
+      max-width: 100%;
+      margin: 10px auto;
+    }
+
+    .tile-type {
+      display: block;
+      margin-bottom: 16px;
```

Comment:
**nitpick:** To maintain uniformity across the Manager, use utility classes from Bootstrap